### ISSUE TITLE
Light bulb incorrectly tries to simplify Nullable<T> in cref doc comments

### ISSUE THREAD

**Issue by brettfo** [dotnet/roslyn#1698]
Repro
-----

``` C#
using System;

/// <summary>
/// <see cref="Nullable{T}"/>
/// </summary>
class C
{
}
```

or

``` VB
Imports System

''' <summary>
''' <see cref="Nullable(Of T)"/>
''' </summary>
Class C
End Class
```

The cref is incorrectly flagged as able to be simplified to `T?` in both languages where it should remain `Nullable{T}` or `Nullable(Of T)`.

**Comment by sharwell** [comment]
Duplicate of #29.

**Comment by srivatsn** [comment]
Closing this one - we'll use #29 to track this issue.
